frontend/api.py

```diff
@@ -40,10 +40,38 @@ def creategame():
 def game():
     return render_template('game.html')
 
-# Loads the scoreboard
+# Retrieves data about each player from the DB, transforms it into a readable format and renders the scoreboard page
+# Implemented by Aidan Roessler from team Vaas
 @app.route('/scoreboard')
 def scoreboard():
-    return render_template('scoreboard.html')
+    # TODO Rotate password and store in an environment variable to not expose it to the FE
+    cnx = mysql.connector.connect(user='webapp', password='masterminds1', host='db', database='MasterMinds')
```

Comment:
Its probably a good idea to store the DB password in an environment variable, especially because this is FE code that is directly exposed to the user